aten/src/ATen/NamedTensorUtils.h

```diff
@@ -1,11 +1,10 @@
 #pragma once
-#ifdef BUILD_NAMEDTENSOR
-
 #include <ATen/NamedTensor.h>
 #include <ATen/core/Tensor.h>
```

Comment:
hmm, looks like this breaks mobile CI because TypeDefault.h will include TensorMethods.h via this path which uses unfinished TypeDefault header (USE_STATIC_DISPATCH=ON for mobile build)...:

In file included from src/pytorch/build_android/aten/src/ATen/TypeDefault.cpp:1:
In file included from aten/src/ATen/TypeDefault.h:12:
In file included from ../aten/src/ATen/NamedTensorUtils.h:3:
In file included from ../aten/src/ATen/core/Tensor.h:12:
../aten/src/ATen/core/TensorMethods.h:64:6: error: use of undeclared identifier 'TypeDefault'

Reply:
Sorry @ljk53, I didn't notice the android build breakage before I landed this. Let me revert this and we can figure out a solution later.

Reply:
Oh, I see you have a fix already. Let's land that first.